### PR TITLE
Adress check return codes extension & major bug fixes (SW-51)

### PR DESCRIPTION
**Pull request by nether-cat**

### CODE REVIEW

File: Frontend/MoptPaymentPayone/plugin.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-        "en": "%%NAME%%"
+        "de": "PAYONE Payment Plugin",
+        "en": "PAYONE Payment Plugin"
     },
     "author": "FATCHIP GmbH",
     "copyright": "Copyright © 2015, FATCHIP GmbH",
     "link": "http://www.fatchip.de/",
     "support": "support@fatchip.de",    
-    "currentVersion": "%%VERSION%%",
+    "currentVersion": "3.4.7",
```
Comment (fjbender) [inline Frontend/MoptPaymentPayone/plugin.json]: The placesholders here should be retained.

Comment (nether-cat) [inline Frontend/MoptPaymentPayone/plugin.json]: File committed accidentally. I have reverted these changes now.

**Comment by nether-cat** [comment]
Commit 556a3e2 fixes several major issues which also affects PR #57 (SW-53)
The two branches were already merged and are up-to-date in this pull request.

**Comment by nether-cat** [comment]
I have added more changes that fix logical issues that have led to multiple address checks. The current head in pull request #57 has also been merged into this branch.

**Pull request merged**